web-api/src/main/java/com/graphhopper/jackson/Jackson.java

```diff
@@ -20,11 +20,16 @@
 import com.bedatadriven.jackson.datatype.jts.JtsModule;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
 
 public class Jackson {
 
     public static ObjectMapper newObjectMapper() {
-        return initObjectMapper(new ObjectMapper());
+        return initObjectMapperWithSnake(new ObjectMapper());
+    }
+
+    public static ObjectMapper initObjectMapperWithSnake(ObjectMapper objectMapper) {
```

Comment:
Using SnakeStrategy to replace some of the mixins also seems like a separate "story"/PR to me. For example I could already use it for the profiles branch if you could split it out and commit this to master already.